app/interactors/compile_github_extension_version_config.rb

```diff
@@ -110,9 +111,19 @@ def read_sha_file(compiled_sha_filename, asset_filename, github_asset_data_hashe
   end
 
   def github_download_url(filename, github_asset_data_hashes_lut)
-    asset_data = github_asset_data_hashes_lut.dig(filename)
```

Comment:
As far as I can tell, the new code is functionally equivalent to the original code.  Plus, when I ran the new code in my sandbox, it didn't solve the problem that when we recompile sensu-pushbullet-handler, the incorrect errors don't go away.

I was able to fix that problem, in commit https://github.com/sensu/bonsai-asset-index/pull/97/commits/fa5073ff07f3d3517644b4a31c351364c55063e5, which I have pushed to this branch.  The bug was that a successful recompilation wasn't clearing the old error messages.

Let's not make this code change in this file, unless you can show that the new code succeeds where the original code fails.

Reply:
Hash.dig was failing to locate the filename in the hash. The combination of Hash.keys resolving keys to a uniform style and Array.include in finding a string resolves a problem I couldn't seem to solve with trying to conform filename.